browserdebuggertools/sockethandler.py

```diff
@@ -179,9 +192,11 @@ def get_events(self, domain, clear=False):
             )
 
         self._flush_messages()
-        events = self._events[domain][:]
+        events = self._events[domain]
         if clear:
             self._events[domain] = []
+        else:
+            events = events[:]
```

Comment:
Is there a reason we are returning a copy of the events to the caller? If so, it would be nice to add a comment here as to why